submissions/rafiatAminu/HTML/html-forms/contact.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
Well done on the work so far

Moving forward, you have to remove the file. It is not supposed to be here. You already have it inside the correct folder. So delete this one